src/main/kotlin/com/github/adeynack/jsoncmp/ComparisonBase.kt

```diff
@@ -0,0 +1,22 @@
+package com.github.adeynack.jsoncmp
+
+import java.io.File
+
+interface ComparisonBase {
+
+    fun compare(arguments: Args)
+
+    fun loadFileContent(filePath: String?, emptyIfNotSpecified: Boolean, fileDescription: String): String =
+        if (filePath == null) {
+            if (emptyIfNotSpecified) {
+                ""
+            } else {
+                throw IllegalArgumentException("No '$fileDescription' file path is specified.")
+            }
+        } else try {
+            File(filePath).readText()
+        } catch (e: Exception) {
```

Comment:
What would you catch? Wider with Throwable or narrower with RuntimeException? Or specifically each and every error I know I could receive here?